cmd/skywire-cli/commands/config/root.go

```diff
@@ -71,6 +71,8 @@ var (
 	minHops                int
 	conf                   *visorconfig.V1
 	usr                    bool
+	ispublic               bool
```

Comment:
Please use camel case (or whatever else its called) so it would be `isPublic` etc.